task/mgmt-gateway/src/update/sp/stub_auxflash.rs

```diff
@@ -0,0 +1,101 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+/// Stub implementation that has the same types and methods as the real
+/// `auxflash` module (our sibling). All method implementations panic: none of
+/// them should ever be called.
+use super::AuxFlash;
+use crate::mgs_handler::UpdateBuffer;
+use gateway_messages::{ResponseError, UpdateId, UpdateStatus};
+
+#[allow(dead_code)] // we never construct any of these variants
```

Comment:
Maybe move these to `auxflash_common.rs`, to avoid duplication between `auxflash.rs` and `stub_auxflash.rs`?

Reply:
Oh, never mind, the important part is `State`, which has a diferent `impl`